src/libcalamaresui/utils/Paste.cpp

```diff
@@ -40,11 +40,15 @@ logFileContents()
         return QByteArray();
     }
     QFileInfo fi( pasteSourceFile );
-    if ( fi.size() > 16_KiB )
+    sizeLimit = ( sizeLimit < 0 ) ? 1024*1024 : sizeLimit * 1024;            //For KiB to bytes
```

Comment:
The documentation says "no limit", while this implementation turns it into 1024KiB.

Reply:
fixed. used `pasteSourceFile.readAll()`